tests/unit/test_qb_client.py

```
@@ -0,0 +1,702 @@
+        resp.raise_for_status = MagicMock()
+
+        client, session = _make_client(resp_sequence_get=[resp])
+        client.get_torrents(category=None, torrent_filter='downloading')
+
+        assert 'category' not in session.get.call_args.kwargs['params']
+        assert session.get.call_args.kwargs['params']['filter'] == 'downloading'
+
+
+class TestGetTorrentsMultipleCategories:
+    def test_aggregates_results(self):
+        r1 = MagicMock(); r1.json.return_value = [{'hash': 'h1'}]; r1.raise_for_status = MagicMock()
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)